lua/modules/crossfire/README.txt

```diff
@@ -0,0 +1,18 @@
+To install this script please download the zip file
+
+crossfire.zip
+
+And install it using the install feature in ethos suite.
+
+Should you choose to install manually, please please place the contents of the zip file inside
+
+/scripts/elrs
+
+You would expect to see files like
+
+/scripts/crossfire/main.lua
+/scripts/crossfire/crossfire.lua
+/scripts/crossfire/compile.lua
+/scripts/crossfire/compiled/
+
+It is important the the folder /scripts/crossfire/compiled exists on your radio for successful byte cache initialisation.
```

Comment:
typo?

Reply:
perhaps we should create the "compiled" directory if it doesn't exist?

Reply:
I can do that - will need to do an update then to match up the same in elrs module.  I will do that as a separate commit.

Reply:
Ok.. so now I remember why I did not make the directory automatically.  I dont think I found any lua functions on ethos that would allow me to!

basically I dont believe we have

- lfs
- os.execute

Any other suggestions?

Reply:
Dont this this can be solved at present - suggest leave as is.